crosswalks/Java (Maven).csv

```diff
@@ -1,30 +1,30 @@
 Property,Java (Maven)
-codeRepository,scm / repositories
+codeRepository,scm
 programmingLanguage,
 review,
 runtimePlatform,
 targetProduct,
 applicationCategory,
 applicationSubCategory,
-downloadUrl,
+downloadUrl,distributionManagement
 fileSize,
 installUrl,
 memoryRequirements,
 operatingSystem,
 permissions,
 processorRequirements,
-releaseNotes,
+releaseNotes,description

```

Comment:
Are you sure? https://maven.apache.org/pom.html#More_Project_Information says `description` is "A short, human readable description of the project."

Reply:
:+1: probably reasonable to leave it out since I think we already map description -> description earlier